rails-base/debian/2.3/Dockerfile

```diff
@@ -4,9 +4,12 @@ RUN apt-get clean && apt-get update \
    && apt-get install -y --no-install-recommends \
       file \
       git \
-      nodejs \
-   && rm -rf /var/lib/apt/lists/* \
-   && ln -s /usr/bin/nodejs /usr/bin/node
```

Comment:
the official package installs `/usr/bin/node` and `/usr/bin/nodejs` which is aliased to `/usr/bin/node` so this line no longer needed